cmd/remote-work-processor/main.go

```diff
@@ -81,23 +82,23 @@ func main() {
 
 	connAttemptChan := make(chan struct{}, 1)
 	connAttemptChan <- struct{}{}
-	var connAttempts uint = 0
+	retryConfig := utils.CreateRetryConfig(opts.RetryInterval, opts.RetryStrategy.Unmarshall(), connAttemptChan)
 
 Loop:
-	for connAttempts < opts.MaxConnRetries {
+	for retryConfig.GetAttempts() < opts.MaxConnRetries {
```

Comment:
Do we have a reason for the MaxConnRetries to not be part of the retry config? Is it used anywhere else?